guides/common/modules/con_synchronizing-content-between-servers.adoc

```diff
@@ -25,5 +25,5 @@ endif::[]
 ifdef::satellite[]
 There are two possible ISS configurations of {Project}, depending on how you deployed your infrastructure.
 Configure your {Project} for ISS as appropriate for your use case scenario.
-For more information, see {InstallingDisconnectedDocURL}how-to-configure-inter-server-sync_{context}[How to Configure Inter-Server Synchronization] in _Installing Satellite Server from a Disconnected Network_.
+For more information, see {InstallingDisconnectedDocURL}how-to-configure-inter-server-sync_{context}[How to Configure Inter-Server Synchronization] in _{InstallingServerDisconnectedDocTitle}_.
```

Comment:
I am unsure if `_{context}` is correct here because it is a DocURL, no xref.

Reply:
It is correct, because the anchor must contain it to work.